component/http/middleware_test.go

```diff
@@ -93,6 +99,7 @@ func TestMiddlewares(t *testing.T) {
 		{"auth middleware false", args{next: handler, mws: []MiddlewareFunc{NewAuthMiddleware(&MockAuthenticator{success: false})}}, 401, "Unauthorized\n"},
 		{"auth middleware error", args{next: handler, mws: []MiddlewareFunc{NewAuthMiddleware(&MockAuthenticator{err: errors.New("auth error")})}}, 500, "Internal Server Error\n"},
 		{"tracing middleware", args{next: handler, mws: []MiddlewareFunc{NewLoggingTracingMiddleware("/index")}}, 202, ""},
+		{"rate limiting middleware", args{next: handler, mws: []MiddlewareFunc{NewRateLimitingMiddleware(getMockLimiter())}}, 202, ""},
```

Comment:
We should make sure that we have a test in place that hits the limit so we can see a different HTTP status code returned. The linter actually has a warning around middleware.go:157.

Reply:
Done